### PR TITLE
document the steps done by the nightly cron jobs generating doc

### PR DESCRIPTION
**Pull request by mgallien**

### CODE REVIEW

**Review by blizzz — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -106,6 +106,19 @@ For developers that want to ease the translation process, please read `this docu
 Building
 --------
 
+Nightly Automated Build Steps
+=============
```
Comment (ChristophWurst) [inline README.rst:110]:
```suggestion
Nightly Automated Build Steps
=============================
```

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Syntax is not rst

File: README.rst [hunk 2]
```diff
@@ -106,6 +106,19 @@ For developers that want to ease the translation process, please read `this docu
 Building
 --------
 
+Nightly Automated Build Steps
+=============================
+
+1. **Fetch sources**
+2. ``git clone https://github.com/nextcloud/documentation.git``
+3. ``cd documentation``
+4. ``git checkout $1``
```
Comment (ChristophWurst) [inline README.rst:115]: is `$1` an argument of a script?

Comment (mgallien) [reply README.rst:115]: this is going to be the name of the target branch
I will modify it to be easier to read

**Review by ChristophWurst — APPROVED** [review]
Syntax looks good now

**Pull request merged**